src/csharp/Grpc.Examples.MathClient/project.json

```diff
@@ -5,6 +5,10 @@
   "configurations": {
     "Debug": {
       "buildOptions": {
+        "define": [ "SIGNED" ],
```

Comment:
For most of the project.json files, the duplicated values are actually generated from a template.